apps/bundle-size/src/commands/build.js

```diff
@@ -0,0 +1,67 @@
+// @ts-check
+
+const chalk = require('chalk');
+const Table = require('cli-table3');
+const fs = require('fs-extra');
+const glob = require('glob');
+const path = require('path');
+const prettyBytes = require('pretty-bytes');
+
+const buildFixture = require('../utils/buildFixture');
+const { hrToSeconds } = require('../utils/helpers');
+const prepareFixture = require('../utils/prepareFixture');
+
+/**
+ * @param {{ verbose: boolean }} options
+ */
+async function build(options) {
+  const { verbose } = options;
+
+  const startTime = process.hrtime();
+  const artifactsDir = path.resolve(process.cwd(), 'dist', 'bundle-size');
+
+  await fs.remove(artifactsDir);
+
+  if (verbose) {
+    console.log(`${chalk.blue('[i]')} artifacts dir is cleared`);
+  }
+
+  const fixtures = glob.sync('bundle-size/*.fixture.js', {
+    cwd: process.cwd(),
+  });
+
+  if (verbose) {
+    console.log(`${chalk.blue('[i]')} Measuring bundle size for ${fixtures.length} fixture(s)...`);
+    console.log(fixtures.map(fixture => `  - ${fixture}`).join('\n'));
+  }
+
+  const preparedFixtures = await Promise.all(fixtures.map(prepareFixture));
+  const measurements = [];
```

Comment:
**suggestion:** from experience I'd avoid writing code that heavily leverages on mutations. 
For explicitness/readability/immutability I'd suggest to use standard functional API that we have in js (.`map` and friends)

Reply:
I will use there `.map()` and `Promise.all`, thanks

Reply:
Done in 270f707.